index.html

```diff
@@ -198,6 +198,29 @@ <h3>Expression Format</h3>
           [=preference=] for the Global Privacy Control value.
         </p>
       </section>
+      <section>
+        <h3>Preference Caching</h3>
+        <p>
+          The [=preference=] MUST be cached on each top-level navigation to ensure consistency in communication of
+          the person's request that their data "not be sold or shared." This means that if the [=preference=] changes
+          during or after a top-level navigation, it WILL NOT be reflected until the next navigation.
+        </p>
+        <p>
+          A [=top-level browsing context=] has a <dfn><code>gpcAtNavigation</code></dfn> boolean.
+          It is initially <code>false</code>.
+        </p>
+        <p>
+          When [=process a navigate fetch=] is run with [=top-level browsing context=] as <code>browsingContext</code>,
```

Comment:
When exactly?  Presumably before anything is sent out.

Overall, I don't think that specifying in this level of detail is necessary.  It would be sufficient to express the requirement that the value be taken at the point at which a navigation is initiated and the value is then permanently associated with the browsing context.

Reply:
I updated the language to mirror the old wording, does this work?